chrome/background.js

```diff
@@ -80,4 +80,51 @@ chrome.commands.onCommand.addListener(function(command) {
       }
     });
   }
+  else if (command == "move-to-new-window") {
+    chrome.permissions.request({
+      permissions: ["tabs"]
+    }, function(granted) {
+      if (granted) {
+        (() => {
```

Comment:
Is this closure needed?

Reply:
Was a remnant from running it's innards in the console. Fixed in b506cd6.